src/utils/constants.js

```diff
@@ -23,25 +23,105 @@ export const rightVocabs = [1, 10];
 export const numberField = [1, 255];
 
 export const contributors = {
-  desktop: [
-    { name: "noctera", description: "25.11.2020" },
-    { name: "luwol03", description: "17.03.2021" },
-    { name: "zikowang", description: "16.03.2021" },
+  frontend: [
+    {
+      name: "noctera",
+      description: "25.11.2020",
+      url: "https://github.com/noctera",
+      imageUrl: "https://github.com/noctera.png",
+    },
+    {
+      name: "luwol03",
+      description: "17.03.2021",
+      url: "https://github.com/luwol03",
+      imageUrl: "https://github.com/luwol03.png",
+    },
+    {
+      name: "zikowang",
+      description: "16.03.2021",
+      url: "https://github.com/zikowang",
+      imageUrl: "https://github.com/zikowang.png",
+    },
+    {
+      name: "Tracer1337",
+      description: "24.04.2022",
+      url: "https://github.com/Tracer1337",
+      imageUrl: "https://github.com/Tracer1337.png",
+    },
   ],
   server: [
-    { name: "noctera", description: "25.11.2020" },
-    { name: "luwol03", description: "16.12.2020" },
-    { name: "retfloww", description: "08.01.2021" },
+    {
+      name: "noctera",
+      description: "25.11.2020",
+      url: "https://github.com/noctera",
+      imageUrl: "https://github.com/noctera.png",
+    },
+    {
+      name: "luwol03",
+      description: "16.12.2020",
+      url: "https://github.com/luwol03",
+      imageUrl: "https://github.com/luwol03.png",
+    },
+    { name: "retfloww", description: `08.01.2021\n-\n10.03.2021` },
```

Comment:
Does this actually work with the \n? Isn't there some css missing?

Reply:
This actually works. But yes there is some css missing. I forgot to copy it from the old responsive branch

Reply:
I think it's something with pre and wrap.